includes/class-sensei-settings.php

```diff
@@ -714,6 +714,42 @@ public function init_fields() {
 			'required'    => 1,
 		);
 
+		$fields['email_reply_to_name'] = [
+			'name'     => __( '"Reply To" Name', 'sensei-lms' ),
+			'type'     => 'input',
+			'default'  => '',
+			'section'  => 'email-notification-settings',
+			'required' => 0,
+		];
+
+		$fields['email_reply_to_address'] = [
+			'name'     => __( '"Reply To" Address', 'sensei-lms' ),
+			'type'     => 'email',
+			'default'  => get_bloginfo( 'admin_email' ),
+			'section'  => 'email-notification-settings',
+			'required' => 0,
+		];
+
+		$fields['email_cc'] = array(
+			'name'          => __( 'CC', 'sensei-lms' ),
+			'description'   => __( 'Enter email addresses to CC on all emails. Separate multiple email addresses with commas.', 'sensei-lms' ),
+			'type'          => 'email_list',
```

Comment:
If we make the type `email` and add the `multiple` attribute, it should work with [multiple email addresses](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email#allowing_multiple_email_addresses).

Reply:
Hum, I'll try it. But I think that produces multiple fields, doesn't it?

Reply:
I found what you meant. Trying to use it.